### ISSUE TITLE
Repo is being taken down due to Copyright issues

### ISSUE THREAD

**Issue by theabbie** [theabbie/torrz#2]
@HanyZaher @tlongren @mahee96

Just letting you know.

**Comment by mahee96** [comment]
@theabbie isn't the api externalized already?

**Comment by theabbie** [comment]
@mahee96 No, it was extracted from their mobile app, I got an email saying that they want it taken down, so, I will, there are other Torrent search APIs as well which are public. Sorry for this.

**Comment by mahee96** [comment]
@theabbie no problem, btw Thanks for your efforts though!!

**Comment by HanyZaher** [comment]
@theabbie thank you
